[2025-09-06T16:32:37.089+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-06T16:32:37.119+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_customers_to_postgres.show_filepath manual__2025-09-06T16:32:36.531127+00:00 [queued]>
[2025-09-06T16:32:37.127+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_customers_to_postgres.show_filepath manual__2025-09-06T16:32:36.531127+00:00 [queued]>
[2025-09-06T16:32:37.127+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-06T16:32:37.136+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): show_filepath> on 2025-09-06 16:32:36.531127+00:00
[2025-09-06T16:32:37.143+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'load_customers_to_postgres', 'show_filepath', 'manual__2025-09-06T16:32:36.531127+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/load_customers_to_postgres.py', '--cfg-path', '/tmp/tmpxunijxss']
[2025-09-06T16:32:37.145+0000] {standard_task_runner.py:91} INFO - Job 58: Subtask show_filepath
[2025-09-06T16:32:37.145+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=3995) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-06T16:32:37.148+0000] {standard_task_runner.py:64} INFO - Started process 3996 to run task
[2025-09-06T16:32:37.180+0000] {task_command.py:426} INFO - Running <TaskInstance: load_customers_to_postgres.show_filepath manual__2025-09-06T16:32:36.531127+00:00 [running]> on host dd8e0e853e24
[2025-09-06T16:32:37.248+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='etl' AIRFLOW_CTX_DAG_ID='load_customers_to_postgres' AIRFLOW_CTX_TASK_ID='show_filepath' AIRFLOW_CTX_EXECUTION_DATE='2025-09-06T16:32:36.531127+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-06T16:32:36.531127+00:00'
[2025-09-06T16:32:37.248+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-06T16:32:37.249+0000] {logging_mixin.py:188} INFO - ðŸ‘€ Looking for file at: /opt/***/data/customer_churn_data.csv
[2025-09-06T16:32:37.249+0000] {python.py:237} INFO - Done. Returned value was: /opt/***/data/customer_churn_data.csv
[2025-09-06T16:32:37.250+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-06T16:32:37.264+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=load_customers_to_postgres, task_id=show_filepath, run_id=manual__2025-09-06T16:32:36.531127+00:00, execution_date=20250906T163236, start_date=20250906T163237, end_date=20250906T163237
[2025-09-06T16:32:37.290+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-06T16:32:37.307+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-06T16:32:37.308+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
